doc/usage.rst

```diff
@@ -89,6 +89,49 @@ Or in Windows, use the little-known BAT file line continuation::
         myscript.spec
 
 
+Running |PyInstaller| with Python optimizations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. Note::
+
+    When using this feature, you should be aware of how the Python bytecode
+    optimization mechanism works. When using ``-O``, ``__debug__`` is set
+    to ``False`` and ``assert`` statements are removed from the bytecode.
+    The ``-OO`` flag additionally removes docstrings.
+
+    Using this feature affects not only your main script, but *all* modules
+    compiled by |PyInstaller|. If your code (or any module imported by your
```

Comment:
This should be "included by PyInstaller", since PyInstaller will not compile the source if there already is a .pyo file.